src/material/progress-bar/_progress-bar-theme.scss

```diff
@@ -3,7 +3,14 @@
 @use '@material/theme/theme-color' as mdc-theme-color;
 @use '@material/linear-progress/linear-progress-theme' as mdc-linear-progress-theme;
 @use 'sass:color';
+@use 'sass:meta';
 
+@function _get-buffer-color($color) {
+  @if (meta.type-of($color) == 'color') {
+    @return color.adjust($color, $alpha: -0.75);
+  }
+  @return $color;
```

Comment:
Any CSS expert help me figure out how to change the alpha channel of this color keeping in mind that it can be any type of color (HSL, RGB, RGBA, hex, etc)?